polkadot-tinkerers.md

```diff
@@ -0,0 +1,104 @@
+The **Tinkerers Track** is for developers with little or no prior blockchain/Polkadot experience who want to experiment, explore, and build something fun during the hackathon.
+
+> 👉 This track is about **creativity, exploration, and fun** — just tinker and build!  
+
+You can build:  
+
+- 🌉 Cross-chain apps  
+- 📊 Data crunching & visualizations  
+- 🎨 UX improvements  
+- 🧪 Proof of Concepts  
+- ✨ Vibe-coded projects  
+
+## 🛠️ Getting Started  
+
+### 🤖 Using AI to Tinker Faster  
+
+You don’t need to know everything — let AI do some heavy lifting:  
+
+- **GitHub Copilot for repos**: Open a repo like [Polkadot SDK](https://github.com/paritytech/polkadot-sdk), and ask Copilot to explain code, functions, or modules. Great for finding answers to “obscure” questions.  
+
+- **Query docs with AI**: Use an AI assistant to query [docs.polkadot.com](https://docs.polkadot.com/) directly — it saves time vs. searching manually.  
+
+### APIs & SDKs  
+- [**Polkadot API (PAPI)**](https://papi.how/) — modern TypeScript API for Polkadot.  
+- [**DeDot**](https://docs.polkadot.com/develop/toolkit/api-libraries/dedot) — lightweight TypeScript library for Polkadot SDK chains, great for quick prototypes.  
+- [**Sidecar**](https://paritytech.github.io/substrate-api-sidecar/) — REST service for blockchain data.  
+- [**Subxt**](https://github.com/paritytech/subxt) — Rust client for Polkadot SDK–based chains.
+- [**Paraspell**](https://paraspell.github.io/docs/)  - Library which makes working with XCM and interoperability on Polkadot easy.
+
+### Data & Indexing  
+- [**SubQuery**](https://subquery.network/) — index and query blockchain data.  
+- [**Polkaholic**](https://polkaholic.io/) — cross-chain explorer & API.  
+- [**DotLake**](https://github.com/paritytech/dotlake-community) — a data ingestion pipeline for Substrate-based blockchains (like Polkadot), view it in action [here](https://data.parity.io).
+
+### Wallets
```

Comment:
Add Novawallet as well. I guess it is okay to include Polkadot JS extension here as these are devs